src/trainers/trainer-list.js

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import trainerService from "./trainer-service"
+// import Button from '@mui/material/Button';
+// import { DataGrid } from '@mui/x-data-grid';
+const { useState, useEffect } = React;
+
+function TrainersList() {
+
+  const [trainers, setTrainers] = useState([])
+  useEffect(() => {
+    findAllTrainers()
+  }, [])
+
+  const findAllTrainers = () =>
+    trainerService.findAllTrainers()
+    .then(trainers => setTrainers(trainers))
```

Comment:
@CookieComputing pls send help

Reply:
look at my comment below for your return value